packages/data-union-solidity/scripts/unichain/deploy-dus.ts

```diff
@@ -0,0 +1,24 @@
+import hhat from "hardhat"
+import { Chains } from "@streamr/config"
+
+const { ethers } = hhat
+
+async function main() {
+    const factorytemplate = await ethers.getContractFactory("DataUnionTemplate")
```

Comment:
I guess we can't have ESLint rule for enforcing camelCase 😅  but we still use write correctCase